### DISCUSSION TITLE
Mindfulness applications

### DISCUSSION THREAD

**Discussion by TailyFair** [InfiniTimeOrg/InfiniTime#1544] [Ideas]
Hi,

Reason I found and ordered my PineTime was to have ability to create my custom mindfulness application for my own use, since InfiniTime is open source.

Idea of application is based on "ideal" breathing tempo - Inhale/Exhale every 4.5 - 5.5 seconds. Application would vibrate every time this interval is reached and notifying user to Inhale/Exhale.

I haven't started development yet, but what others think about including this or different mindfulness applications to InfiniTime?

P.S. Articles regarding slow breathing benefits:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5709795/
https://www.theguardian.com/lifeandstyle/2020/jul/26/every-breath-you-take-the-lost-art-of-breathing
https://www.moonbird.life/magazine/slow-breathing-for-health/

**Comment by JPeisach** [comment]
This is a great idea!

The 4-7-8 breathing would be popular, and I would likely use it. Some UI along the way would be cool too (like the heart rate app but it would countdown the timer, change title and color depending on the current step you're on.

Slightly off topic but in the idea of being 'mindful' of work, and thinking of time, a Pomodoro timer wouldn't be hard to implement.

This could go in an app of its own, but I think it might be easier to implement it inside the Timer app by making it a scrollable view.

**Comment by leroivi** [comment]
Hi, I was thinking of a pomodoro app as a first simple application to get my hands on.
The pomodoro app would feature 2 timers, one for work session and one for pauses session. During the work session, the pomodoro app would turn the device into DoNotDisturb mode, then it would turn the device back in its previous mode at the pause.
I think the device should ring at the end of each phase and wait the user confirmation to proceed the next timer.
